### PR TITLE
Added license badge and missing punctuation

### PR DESCRIPTION
**Pull request by sumitkharche**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,28 +4,29 @@ API Platform is a next-generation web framework designed to easily create API-fi
...
+* Create specs and tests with a **developer friendly API testing tool** on top of [Behat](https://behat.org/).
 
 [![Travis CI](https://travis-ci.org/api-platform/core.svg?branch=master)](https://travis-ci.org/api-platform/core)
 [![CircleCI](https://circleci.com/gh/api-platform/core/tree/master.svg?style=shield)](https://circleci.com/gh/api-platform/core/tree/master)
 [![AppVeyor](https://ci.appveyor.com/api/projects/status/grwuyprts3wdqx5l/branch/master?svg=true)](https://ci.appveyor.com/project/dunglas/dunglasapibundle/branch/master)
 [![Codecov](https://codecov.io/gh/api-platform/core/branch/master/graph/badge.svg)](https://codecov.io/gh/api-platform/core/branch/master)
 [![SymfonyInsight](https://insight.symfony.com/projects/92d78899-946c-4282-89a3-ac92344f9a93/mini.svg)](https://insight.symfony.com/projects/92d78899-946c-4282-89a3-ac92344f9a93)
 [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/api-platform/core/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/api-platform/core/?branch=master)
+[![GNU license](https://img.shields.io/github/license/api-platform/api-platform)](https://github.com/api-platform/api-platform/blob/master/LICENSE)
```
Comment (dunglas) [inline README.md]: API Platform is MIT licensed, not GNU. And as GitHub already displays the license, we don't add a badge for it. Can you remove this line please?

Comment (sumitkharche) [reply README.md]: @dunglas I will remove it.

Comment (sumitkharche) [reply README.md]: @dunglas I have removed license badge

**Comment by dunglas** [comment]
Thanks @sumitkharche

**Pull request merged**